gustaf/io/meshio.py

```diff
@@ -72,18 +72,65 @@ def load(fname):
     return mesh
 
 
-def export(mesh, fname):
-    """Currently not implemented function.
+def export(mesh, fname, submeshes=None):
+    """Export mesh elements and vertex data into meshio and use its write
+    function. The definition of submeshes with identical vertex coordinates
+    is possible. In that case vertex numbering and data from the main mesh
+    are used.
```

Comment:
Please provide an example here by using the 
```
.. code-block:: python 
```
syntax
and also link to meshio's github repo  
```suggestion
    are used. For more export options, refer to meshio's documentation [link]()
```

Reply:
https://github.com/tataratat/splinepy/blob/7051e0bc1fea144f06dcbcc37f4681ebebfb0fb8/splinepy/helpme/integrate.py#L110 
Here is an example for a code-block. It will look [like](https://tataratat.github.io/splinepy/_generated/splinepy.helpme.integrate.Integrator.html#splinepy.helpme.integrate.Integrator) this in the built docs: